modules/cms-kit/src/Volo.CmsKit.Public.Application.Contracts/Volo/CmsKit/Public/Tags/ITagAppService.cs

```diff
@@ -7,6 +7,6 @@ namespace Volo.CmsKit.Public.Tags
 {
     public interface ITagAppService : IApplicationService
     {
-        Task<List<TagDto>> GetAllRelatedTagsAsync(GetRelatedTagsInput input);
+        Task<List<TagDto>> GetAllRelatedTagsAsync(string entityType, string entityId);
```

Comment:
I'm not sure of this changes, as @cotur 's say each input and output parameters should be object. That's why we created 'GetRelatedTagsInput' type.

Reply:
All other application services and controllers use the same convention. That's why I've made the changes. If those implementations are against the convention we can change the other usages.

Reply:
Ok, it's better now. 👍